src/NoFrixion.MoneyMoov/Models/Account/PaymentAccount.cs

```diff
@@ -132,4 +132,19 @@ public string Summary
     /// Externally connected account can be used to view account balances and transactions. 
     /// </summary>
     public bool IsConnectedAccount { get; set; }
+
+    /// <summary>
+    /// The Icon for external accounts
+    /// </summary>
+    public string ExternalAccountIcon { get; set; }
+
+    /// <summary>
+    /// The bank name for external accounts
+    /// </summary>
+    public string BankName { get; set; }
+    
+    /// <summary>
+    /// The date that the external account will expire
+    /// </summary>
+    public DateTimeOffset? ExpiryDate { get; set; }

```

Comment:
Will the expiry date be the same as the consent expiry?

Reply:
Yes